### PR TITLE
[Example]update NER example readme on module prediction

### PR DESCRIPTION
**Pull request by roywei**
## Description ##
Fix https://github.com/apache/incubator-mxnet/issues/11859
Added an inference section in README to point to the right example on how to do prediction using BucketingModule

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- Update README

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: example/named_entity_recognition/README.md [hunk 1]
```diff
@@ -15,4 +15,11 @@ To reproduce the preprocessed training data:
...
-- `$ cd src && python ner.py`
\ No newline at end of file
+- `$ cd src && python ner.py`
+
+To inference using trained model:
+
+1. Re-create the bucketing module using `sym_gen` defined in `ner.py`
+2. Loading saved parameters using `module.set_params()`
+
+Refere to [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/e9a590fa6554231fba404dad08acee5cd3e786a8/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167) and this [issue](https://github.com/apache/incubator-mxnet/issues/5008) on Bucketing Module Prediction
```
Comment (piyushghai) [inline example/named_entity_recognition/README.md]: Nit : Typo in 'Refere'.

**Comment by anirudhacharya** [comment]
@mxnet-label-bot [pr-awaiting-review, Example]

File: example/named_entity_recognition/README.md [hunk 2]
```diff
@@ -15,4 +15,11 @@ To reproduce the preprocessed training data:
...
-- `$ cd src && python ner.py`
\ No newline at end of file
+- `$ cd src && python ner.py`
+
+To inference using trained model:
+
+1. Re-create the bucketing module using `sym_gen` defined in `ner.py`
+2. Loading saved parameters using `module.set_params()`
+
+Refer to [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/e9a590fa6554231fba404dad08acee5cd3e786a8/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167) and this [issue](https://github.com/apache/incubator-mxnet/issues/5008) on Bucketing Module Prediction
```
Comment (nswamy) [inline example/named_entity_recognition/README.md]: can you reference this relative to the master rather than a particular commit?

**Comment by roywei** [comment]
@nswamy addressed comments. Please help merge it if looks good. Thanks!

File: example/named_entity_recognition/README.md [hunk 3]
```diff
@@ -15,4 +15,12 @@ To reproduce the preprocessed training data:
 
 To train the model:
 
-- `$ cd src && python ner.py`
\ No newline at end of file
+- `$ cd src && python ner.py`
+
+To inference using trained model:
```
Comment (aaronmarkham) [inline example/named_entity_recognition/README.md]:
```suggestion
To run inference using trained model:
```

File: example/named_entity_recognition/README.md [hunk 4]
```diff
@@ -15,4 +15,12 @@ To reproduce the preprocessed training data:
...
+
+1. Re-create the bucketing module using `sym_gen` defined in `ner.py`
```
Comment (aaronmarkham) [inline example/named_entity_recognition/README.md]:
```suggestion
1. Recreate the bucketing module using `sym_gen` as defined in `ner.py`
```

File: example/named_entity_recognition/README.md [hunk 5]
```diff
@@ -15,4 +15,12 @@ To reproduce the preprocessed training data:
...
+2. Loading saved parameters using `module.set_params()`
+
+Refer to [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167)
```
Comment (aaronmarkham) [inline example/named_entity_recognition/README.md]:
```suggestion
Refer to the [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167)
```

**Comment by kalyc** [comment]
@mxnet-label-bot update [pr-awaiting-merge]

**Comment by kalyc** [comment]
@piyushghai @nswamy PR looks good to merge to me - could you review the updated diff?

File: example/named_entity_recognition/README.md [hunk 6]
```diff
@@ -15,4 +15,12 @@ To reproduce the preprocessed training data:
...
-- `$ cd src && python ner.py`
\ No newline at end of file
+- `$ cd src && python ner.py`
+
+To run inference using trained model:
+
+1. Recreate the bucketing module using `sym_gen` defined in `ner.py`
+2. Loading saved parameters using `module.set_params()`
+
+Refer to the [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167)
```
Comment (sandeep-krishnamurthy) [inline example/named_entity_recognition/README.md]: Line numbers usually get obsolete and I have seen many broken links due to that. Can we just refer to cudnn_rnn_bucketing.py example and provide function name that users should read.

Comment (roywei) [reply example/named_entity_recognition/README.md]: Thanks for pointing out, addressed this

**Pull request merged**